docs/getting-started.md

```diff
@@ -13,28 +13,28 @@ Powsybl provides an internal complete grid model with substations, voltage level
 two and three windings transformers, batteries, generators, loads, shunt and static VAR compensators, etc. The grid model can be
 extended with extensions to complete the modelling of the equipments (dynamic profile, short-circuit profile, monitoring, etc.). It also provides importers and exporters for several common pan-european exchange formats ([Entso-E CIM/CGMES](https://www.entsoe.eu/digital/common-information-model/cim-for-grid-models-exchange/), [UCTE-DEF](https://cimug.ucaiug.org/Groups/Model%20Exchange/UCTE-format.pdf), etc.).
 
-Powsybl library provides several API for power systems' simulations and analysis (power flow computations, security analysis,
+Powsybl as a library provides several APIs for power systems' simulations and analyses (power flow computations, security analyses,
```

Comment:
What is the difference between *analysis* and *analyse*? Should we change the name of the tool?

Reply:
analysis is singular, analyses is plural